app/models/event.rb

```diff
@@ -18,7 +18,7 @@ def description
   end
 
   def location
-    @data['location']
+    @data['location'] || ''
```

Comment:
This and its pair in `#attachments` won’t be necessary if you check for `blank?` in the view. That way, `Event` can be concerned strictly with understanding Google Calendar Event API responses, and not formatting.